careers/blocks/header/header.css

```diff
@@ -605,14 +606,22 @@ header nav .nav-tools p:first-of-type a:hover {
     transition: unset;
   }
 
-  header nav .nav-sections .local-nav ul li:first-of-type a:not(.more-link, .menu-item)::after,
-  header nav .nav-sections .local-nav ul li a:hover::after {
+  header nav .nav-sections .local-nav ul li a::after {
     content: '';
     display: block;
+    position: absolute;
+    width: 100%;
     height: 3px;
-    transition: transform .15s cubic-bezier(0.4, 0, 0.2, 1);
     background: var(--static-color-white);
-    margin: auto 0;
+    left: 0;
+    bottom: -10px;
+    transform: scaleX(0);
+    transition: transform .15s cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  header nav .nav-sections .local-nav ul li a.active::after,
+  header nav .nav-sections .local-nav ul li a:hover::after {
+    transform: scaleX(1);
```

Comment:
Question: Why is this needed? What does it do?

Reply:
It basically says the we moved from scaleX(0) to scaleX(1) on hover - which means that white block under the nav items will grow from 0 to 1 horizontally (X axis) which gives that cool animation effect